### ISSUE TITLE
Add to the GPS widget to show the coordinate in PLUS CODES

### ISSUE THREAD

**Issue by sdellava** [EdgeTX/edgetx#4882]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

This request try to simplify the usage of GPS coordinates collected via telemetry.

There are many coordinate formats but, to get the best precision, all of them requre to copy more than 15 number and insert them in a mapping tool.

The Plus Codes (https://maps.google.com/pluscodes/technology/) reduce a complete coordinate up to 6 characters. 

The full coordinate is made of 11 characters but the first five identify a very large quadrant.

### Describe the solution you'd like

I guess the simple solution is to improve the GPS widget to allow to choose the dd°ssss' format or the Plus Code format.  

Allowing the page to collect the device position, the last 6 charters of the plus codes mark can be inserted here: https://plus.codes/map

### Additional context

There are others feature requests related to the GPS format but no one suggest to use Plus Codes or ask to show the coordinate in a QRcode format. 

All are good alternatives, but the solution I've suggested is the simplest to implement a provide a good level of optimization.

**Comment by 3djc** [comment]
Have you read comments in: https://github.com/EdgeTX/edgetx/issues/4614

**Comment by sdellava** [comment]
> Have you read comments in: #4614 ? They might be addressing your needs

Yes. My point is to allow the GPS widget to be more flexible. Using a LUA is an good option, but the widgets allow the user to setup his preferred layout.

**Comment by 3djc** [comment]
Most widgets ARE lua..

**Comment by pfeerick** [comment]
I think you missed JCs point... these are already widgets (just Lua based, rather than built into the firmware - meaning you just need to install it to the SD card) which already provide this functionality. i.e. https://github.com/kristjanbjarni/opentx-widgets gives you a plus code.

**Comment by sdellava** [comment]
Ohh. Tnx, yes I totally missed the point.

Il mar 16 apr 2024, 19:20 Peter Feerick ***@***.***> ha
scritto:
